# Trivial little patch to silence a build warning.  The Linux kernel searches
# for a command at an absolute path, then runs it out of $PATH, but for us
# /bin isn't necessarily in $PATH.

diff -ru linux.bak/scripts/mkcompile_h linux/scripts/mkcompile_h
--- linux.bak/scripts/mkcompile_h	2009-12-05 15:06:52.000000000 -0600
+++ linux/scripts/mkcompile_h	2009-12-05 15:15:37.000000000 -0600
@@ -67,9 +67,9 @@
   echo \#define LINUX_COMPILE_BY \"`whoami`\"
   echo \#define LINUX_COMPILE_HOST \"`hostname | $UTS_TRUNCATE`\"
 
-  if [ -x /bin/dnsdomainname ]; then
+  if [ ! -z "$(which dnsdomainname)" ]; then
     domain=`dnsdomainname 2> /dev/null`
-  elif [ -x /bin/domainname ]; then
+  elif [ ! -z "$(which domainname)" ]; then
     domain=`domainname 2> /dev/null`
   fi
 
